### ISSUE TITLE
add `bundle which <GEM>` command

### ISSUE THREAD

**Issue by jethrodaniel** [rubygems/bundler#7652]
We already have `bundle open <GEM>`, but sometimes I just want the full path to an installed gem, instead of opening a new `$EDITOR`.

I think we should add this.

Example:

```
$ bundle which racc
/home/user/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/racc-1.4.16
```

**Comment by deivid-rodriguez** [comment]
I very much agree with this feature request, since I've missed it myself too.

**Comment by colby-swandale** [comment]
`bundle path` has this feature.

    $ bundle info rails --path
    /Users/c/.asdf/installs/ruby/2.6.5/lib/ruby/gems/2.6.0/gems/rails-6.0.2.1

**Comment by deivid-rodriguez** [comment]
RIght, I somehow managed to forget about that. Do you see value on a separate CLI command, or you'd rather close this?

**Comment by jethrodaniel** [comment]
I like the idea of a separate `which` command (I'd use it more), but **not** the amount of code required for it when we already have that functionality elsewhere.

Does bundler have any future plans for internal aliases, similar to git aliases? More specifically, it'd be cool if `bundle which <gem>` simply was an alias for `bundle info <gem> --path`.

I'd rather not have the `which` command if it requires duplication of existing code.

**Comment by colby-swandale** [comment]
I don't personally think there is value in a separate command, but i could be completely wrong. My only suggestion is that i would more prefer the command to be `bundle where` instead, `bundle which` doesn't feel right IMO.

**Comment by bmorrall** [comment]
Unless I'm missing something there is already a `bundle show` command that lists the path to a gem?!

```
$ bundle show parser
/Users/me/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/parser-2.7.0.4
```
